SparkPost/matchbox
# UX-261 Create usePrefersReducedMotion hook

### What Changed

- Adds usePrefersReducedMotion hook

### How To Test or Verify

- `npm run start:storybook`
- Visit [http://localhost:9001/?path=/story/utility-useprefersreducedmotion--example-usage](usePrefersReducedMotion story)
- Verify that value is either `no-preference` or `reduce` based on your settings
- If on a mac, you can change this setting by changing `System Preferences` > `Accessibility` > `Display` > `Reduce motion`

### PR Checklist
